deploy/Jenkinsfile

```diff
@@ -6,7 +6,7 @@
       }
 
     parameters {
```

Comment:
choice(name: 'inventory', choices: ['ci', 'dev', 'sit'],  description: 'This is the inventory file for the environment to deploy configuration')